openshift/e2e-common.sh

```diff
@@ -114,7 +114,9 @@ function install_knative(){
   sed -i -e 's/kourier-control.knative-serving/kourier-control.knative-serving-ingress/g'  third_party/kourier-latest/kourier.yaml
 
   sed -i -e "s|docker.io/maistra/proxyv2-ubi8:.*|${KOURIER_GATEWAY}|g"                                         ${CATALOG_SOURCE}
-  sed -i -e "s|registry.svc.ci.openshift.org/openshift/knative-.*:kourier|${KOURIER_CONTROL}|g"               ${CATALOG_SOURCE}
+
+  # TODO: Upstream kourier build doesn't run on current OCP 4.5, https://bugzilla.redhat.com/show_bug.cgi?id=1934177
```

Comment:
Is it the right Bugzilla? It mentions Knative Camel and newer versions of Openshift.

Reply:
kind of, yes. The regression was backported to 4.5 and 4.6 and it affects any image based on gcr.io/distroless/static:nonroot  

anyway, it is the bz that tracks the fix (and also mentions the intention to backport to 4.5), see

See https://bugzilla.redhat.com/show_bug.cgi?id=1934177#c13

Reply:
ok. thanks